backend/src/test/java/heartbeat/service/jira/JiraServiceTest.java

```diff
@@ -816,32 +819,39 @@ void shouldGetRealDoneCardWhenCallGetStoryPointsAndCycleTimeWhenUseHistoricalAss
 		when(urlGenerator.getUri(any())).thenReturn(baseUrl);
 		when(jiraFeignClient.getJiraCards(any(), any(), anyInt(), anyInt(), any(), any())).thenReturn(allDoneCards);
 		when(jiraFeignClient.getJiraCardHistory(baseUrl, "ADM-475", token))
-			.thenReturn(CARD1_HISTORY_FOR_HISTORICAL_ASSIGNEE_FILTER_METHOD().build());
+			.thenReturn(CARD1_HISTORY_FOR_HISTORICAL_ASSIGNEE_FILTER().build());
```

Comment:
given a `new line` between `given when then`

Reply:
Already update this pattern.